### DISCUSSION TITLE
NFS mounted share in mounted volume... continuation of discussion #20745

### DISCUSSION THREAD

**Discussion by PelleFK** [containers/podman#20760] [Q&A]
I have a Apache running in a container via Podman. 
I have one NFS share mounted in a subfolder to a mounted folder via volumes. It is muonted like:
 - ./html:/var/www/html/
 without trailing :Z or :z
 The NFS share are mounted at html/shared.
 Now I have the problem thar my web server cannot write to any files in html-folder, not in the NFS mounted "shared" either. If I remove the NFS share the web server can wright to files (some pages write to some local Json files).
 I'm using Redhat 8 so I guess SELinux are involved in the problem. 
 
"ls -laZ html/" gives among others
```
drwxrwxrwx.  4 LOCAL_USER LOCLA_GROUP system_u:object_r:nfs_t:s0       4096 Nov 23 11:41 shared
drwxr-xr-x.  2 LOCAL_USER LOCLA_GROUP unconfined_u:object_r:usr_t:s0   4096 Nov 23 11:48 info
```

I guess it is due to the SELinux I cannot write to she shared folder but it is also disable writing to info folder.

How can I solve this?

**Comment by rhatdan** [comment]
Are you sure this is an selinux issue?  If you do `setenforce 0` does it work or not.  If it works then it is not an SELinux.

Are you running rootless containers?
